sdk/communication/Azure.Communication.Rooms/src/Models/ParticipantsCollection.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.Communication.Rooms
+{
+    /// <summary> Collection of participants who belong to a room. </summary>
+    public partial class ParticipantsCollection
+    {
+        /// <summary> Initializes a new instance of ParticipantsCollection. </summary>
+        /// <param name="participants"> Room Participants. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="participants"/> is null. </exception>
+        internal ParticipantsCollection(IEnumerable<RoomParticipant> participants)
+        {
+            if (participants == null)
+            {
+                throw new ArgumentNullException(nameof(participants));
+            }
```

Comment:
Nit - prefer `Argument.AssertNotNull(participants, nameof(participants))` for these.

Reply:
Thanks. Addressed.